manifests/plugin/powerdns/recursor.pp

```diff
@@ -0,0 +1,19 @@
+define collectd::plugin::powerdns::recursor (
+  $ensure    = 'present',
+  Optional[String] $socket = undef,
+  Array[String] $collect   = [],
```

Comment:
please align the = chars. For all the strings you could also do something like `Optional[String[1]]` which enforces a minimal string length of 1, so `''` isn't valid

Reply:
done